go.mod

```diff
@@ -4,11 +4,11 @@ go 1.17
 
 require (
 	github.com/gorilla/websocket v1.5.0 // indirect
-	github.com/mattermost/mattermost-server/v6 v6.4.0
+	github.com/mattermost/mattermost-server/v6 v6.6.0
```

Comment:
Will this require us to bump the minimum server?

Reply:
Nope, changes are backwards compatible so while the reliable send will only work on 6.5 onwards previous versions shouldn't be affected.